rts/lra/test/arquillian-extension/src/main/java/io/narayana/lra/arquillian/AppServerCoordinatorDeploymentObserver.java

```diff
@@ -53,15 +53,15 @@
      * which is deployed when the app server starts.
      */
     public void handleAfterStartup(@Observes AfterStart event, Container container) throws Exception {
-        if(!container.getName().contains(CONTAINER_NAME_RECOGNITION)) {
+        if (!container.getName().contains(CONTAINER_NAME_RECOGNITION)) {
```

Comment:
It is clear that `String LRA_COORDINATOR_DEPLOYMENT_NAME = "lra-coordinator";` is built into the deployment.
But it is not clear how `String CONTAINER_NAME_RECOGNITION = "as-coordinator";` is set and used (for example it is not present in `arquillian-wildfly.xml`).

Reply:
There was a bit of text in the definition of the field. I've extended it a little bit now.